backup_and_restore/application_backup_and_restore/oadp-advanced-topics.adoc

```diff
@@ -0,0 +1,18 @@
+:_content-type: ASSEMBLY
+[id="oadp-advanced-topics"]
+= Advanced OADP features and functionalities
+include::_attributes/common-attributes.adoc[]
+:context: oadp-advanced-topics
+:oadp-advanced-topics:
+
+toc::[]
+
+This document provides information on advanced features and functionalities of OpenShift API for Data Protection (OADP).
+
+[id="oadp-different-kubernetes-api-versions_{context}"]
```

Comment:
[Anchors](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-in-assembly-files) in assemblies do not use the `_{context}` element.